tensilelite/Tensile/Tests/common/gemm/dtva_adv.yaml

```diff
@@ -0,0 +1,204 @@
+TestParameters:
+  marks: [skip-gfx900, skip-gfx906, skip-gfx908, skip-gfx90a, skip-gfx1010, skip-gfx1011, skip-gfx1012, skip-gfx1030, skip-gfx1100, skip-gfx1101, skip-gfx1102, skip-gfx1200, skip-gfx1201] # not supported by arch
+
+GlobalParameters:
+  NumElementsToValidate: -1
+  MinimumRequiredVersion: 4.14.0
+  PrintLevel: 1
+  Device: 0
+  MergeFiles: False
+  KernelTime: True
+  MaxWorkspaceSize: 13421772800
+  DataInitTypeAlpha: 1
+  DataInitTypeBeta: 1
+  BoundsCheck: 2
+  #MaxFileName: 256
+
+BenchmarkProblems:
+  ########################################
+  # HHS TN DTVA + BIAS + SAV + Activation
+  # Check LDS usage of Epilogues
+  ########################################
+  -
+    - # ProblemType
+      OperationType: GEMM
```

Comment:
The first problem set shows the validation failure when LDS-of-AB < LDS-of-epilogue.